hydra-cluster/hydra-cluster.cabal

```diff
@@ -262,31 +262,35 @@ benchmark bench-e2e
   main-is:            Main.hs
   type:               exitcode-stdio-1.0
   other-modules:      Bench.EndToEnd
+                    , Bench.Options
+                    , Bench.Summary
   build-depends:
-    , aeson
-    , base                  >=4.7 && <5
-    , bytestring
-    , cardano-crypto-class
-    , containers
-    , directory
-    , filepath
-    , hspec
-    , HUnit
-    , hydra-cardano-api
-    , hydra-cluster
-    , hydra-node
-    , hydra-prelude
-    , hydra-test-utils
-    , io-classes
-    , lens
-    , lens-aeson
-    , optparse-applicative
-    , process
-    , QuickCheck
-    , regex-tdfa
-    , scientific
-    , strict-containers
-    , time
+                    HUnit
```

Comment:
Seems like your cabal file parser is misbehaving.

Reply:
Crap